fbchat/models.py

```diff
@@ -37,14 +37,17 @@ class Thread(object):
     last_message_timestamp = None
     #: Number of messages in the thread
     message_count = None
-    def __init__(self, _type, uid, photo=None, name=None, last_message_timestamp=None, message_count=None):
+    #: List of :class:`Event`
+    event_reminders = None
+    def __init__(self, _type, uid, photo=None, name=None, last_message_timestamp=None, message_count=None, event_reminders=[]):
```

Comment:
See [this discussion](https://stackoverflow.com/q/1132941) on mutable default arguments. In short, `event_reminders` should be set to `None` by default, and initialized to a list when the instance is created

Reply:
[This](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) is also a good resource

Reply:
Wow, I didn't know that. Thank you!